config/env_schema.hjson

```diff
@@ -57,6 +57,13 @@
                 # Canvas API
                 "CANVAS_URL": {"type": "string"},
                 "CANVAS_TOKEN": {"type": "string"},
+                "SUPPORTED_LTI_TOOLS" : {
+                    type: "object"
+                    "properties": {
+                        "name": {"type": "string"}
+                        "id": {"type": "integer"}
+                    }
+        }
```

Comment:
Nit-pick: Please fix indentation; this `{` should be under the beginning of `SUPPORTED_LTI_TOOLS`.

Reply:
Sorry, I'd provide a suggestion but spaces and tabs are weird in the GitHub UI.

Reply:
Yeah, weird!